documentation/src/docs/state-synchronizer-design.md

```diff
@@ -7,24 +7,27 @@ You may obtain a copy of the License at
 
     http://www.apache.org/licenses/LICENSE-2.0
 -->
-# StateSynchronizer Design
+# State Synchronizer Design
 
-A StateSynchronizer provides a means for data to be written and read by multiple processes, while consistency is guaranteed using optimistic checks. A rough version of this API is [checked in here.](https://github.com/pravega/pravega/blob/master/client/src/main/java/io/pravega/client/state/StateSynchronizer.java)
+A State Synchronizer provides means for data to be written and read by multiple processes, while consistency is guaranteed using optimistic checks. For more information, check out the [State Synchronizer API]. (https://github.com/pravega/pravega/blob/master/client/src/main/java/io/pravega/client/state/StateSynchronizer.java)
```

Comment:
There is a problem with this link, as the whole link appears in the page. Probably there is an extra space (plus a ' .' ) between the test to be linked and the link.

Reply:
@RaulGracia edited